src/main/java/core/basesyntax/figures/Rectangle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Rectangle extends Figure {
+    private static final String type = "rectangle";
+    private final int width;
+    private final int height;
+
+
+    public Rectangle(int width, int height, String color) {
+        this.width = width;
+        this.height = height;
+        this.color = color;
```

Comment:
It's good practice to have a super() call to initialize fields inherited from the superclass, even if it's the default constructor. This ensures that the Figure class is properly set up.